src/common.h

```diff
@@ -0,0 +1,4 @@
+
+#if defined(DEBUG) || !defined(NDEBUG)
+#include "readies/cetara/diag/gdb.h"
```

Comment:
also, wouldn't angle-bracket includes be more appropriate?

Reply:
Well, it's kind of religious. I tend to use <> with for system (sysroot) SDKs. But I'll follow the local convention.